src/integration/data/query.test.ts

```diff
@@ -12,9 +12,10 @@ describe('query', () => {
     index: Index,
     ns: Index,
     namespace: string,
-    recordIds: string[];
+    recordIds: string[],
+    numberOfRecords: number;
 
-  beforeEach(async () => {
+  beforeAll(async () => {
```

Comment:
The primary changes to this file are:

- Only seeding record data in the `beforeAll` and using those records across tests rather than deleting and re-upserting every test, which was time consuming and prone to flake.
- Fixing `query with includeValues: true` as it wasn't properly upserting before deleting (whoops). This passed before but I think we started returning 404s for deletes on non-existing records which broke things.